guides/v2.1/contributor-guide/backward-compatible-development/index.md

```diff
@@ -189,6 +189,21 @@ Do not remove or rename {% glossarytooltip c57aef7c-97b4-4b2b-a999-8001accef1fe
 Do not change argument types.
 Instead of changing argument name or type, introduce new event argument with new name or type and deprecate the old argument by adding `@deprecated` annotation before dispatching the event.
 
+
+{% collapsible Example Code %} {% highlight php startinline %} /**
+
+$transportObject = new DataObject($transport);
+
+    /**
+     * Event argument `transport` is @deprecated. Use `transportObject` instead.
+     */
+    $this->eventManager->dispatch(
+        'email_invoice_set_template_vars_before',
+        ['sender' => $this, 'transport' => $transportObject->getData(), 'transportObject' => $transportObject]
```

Comment:
I suggest to use underscore for parameter names this is seem to be the convention used in most places.

Reply:
Array indexes (transport, transportObject). We can add it as a suggestion